Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "ykmir"
+version = "0.1.0"
+authors = ["Edd Barrett <vext01@gmail.com>"]
+edition = "2018"
+
+[dependencies]
+serde = "1.0"
+serde_derive = "1.0"
+fallible-iterator = "0.1"
+
+# We are using a git version to work around a bug:
+# https://github.com/3Hren/msgpack-rust/issues/183
+[dependencies.rmp-serde]
```

Comment:
If you want you can do something like `rmp-serde = { git = "...", rev = "..." }`. I have no preferences about which is better.

Reply:
Yeah, I started with that, but it makes a very long time. I prefer this way.